ouroboros-consensus/test/storage-test/Main.hs

```diff
@@ -1,11 +1,19 @@
+{-# LANGUAGE NumericUnderscores #-}
 module Main (main) where
 
+import           Control.Concurrent (threadDelay)
+import           Control.Concurrent.Async (race_)
+import           Control.Monad (forever)
+import           System.IO (hFlush, stdout)
 import qualified Test.Ouroboros.Storage
 import           Test.Tasty
 import           Test.Util.TestEnv
 
 main :: IO ()
-main = defaultMainWithTestEnv defaultTestEnvConfig tests
+main = runTests `race_` heartbeat
+  where
+    runTests  = defaultMainWithTestEnv defaultTestEnvConfig tests
+    heartbeat = forever $ threadDelay (30 * 1_000_000)  >> putChar '.' >> hFlush stdout
```

Comment:
What is taking so long that we need this? (I suppose that it's for Hydra?)

Reply:
Removing it for now. Let's see if Hydra complains